projects/storefrontlib/cms-components/misc/site-context-selector/site-context-selector.component.ts

```diff
@@ -17,7 +16,6 @@ import { SiteContextType } from './site-context.model';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class SiteContextSelectorComponent {
-  siteContextService: SiteContext<any>;
```

Comment:
Would this be a breaking change?

Reply:
We might not use it but customers could, it's highly unlikely but still possible.